tests/unit/FileExports/CustomerFileExportTest.php

```diff
@@ -14,6 +17,35 @@ public function getFileExportClass(): string
         return CustomerFileExport::class;
     }
 
+    public function testUserCreateByRoles()
```

Comment:
it's better to use a data provider for the roles

Reply:
added data provier for test